projects/mupen64plus.md

```diff
@@ -0,0 +1,18 @@
+# Mupen64Plus and mupen64plus-ui-python
```

Comment:
move to `projects/python/mupen64plus.md`

Reply:
Done!